dictionaries/CallMap_80_delta.php

```diff
@@ -55,6 +55,7 @@
 'DateTimeImmutable::format' => ['string', 'format'=>'string'],
 'DateTimeImmutable::createFromInterface' => ['self', 'object' => 'DateTimeInterface'],
 'DateTimeZone::listIdentifiers' => ['list<string>', 'timezoneGroup='=>'int', 'countryCode='=>'string|null'],
+'date' => ['string', 'format'=>'string', 'timestamp='=>'?int'],
```

Comment:
In addition to this, this entry should be in `CallMap.php`. Also, the old signature needs to be present in 'old' section.

Reply:
Added the old signature to this file: https://github.com/vimeo/psalm/pull/6244/commits/81661395ce952b9a66c595db2ada8019aeab7709
Added the new signature to CallMap: https://github.com/vimeo/psalm/pull/6244/commits/b2ba774cb9b08efa16e2d1ccc870911c8939408d